core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -565,6 +566,29 @@ static function (array $attributes): array {
         ;
     }
 
+    private function addBackupNode(): NodeDefinition
+    {
+        return (new TreeBuilder('backup'))
+            ->getRootNode()
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->scalarNode('backup_dir')
```

Comment:
```suggestion
                ->scalarNode('target_dir')
```

Reply:
I don't like that so I won't use that suggestion.